module/apmecho/go.mod

```diff
@@ -1,17 +1,13 @@
-module go.elastic.co/apm/module/apmecho
```

Comment:
The changes to the existing go.mod and go.sum can be reverted now, right?

Reply:
Yes, I forgot that.

Reply:
go.mod and go.sum are still modified, can you please remove the changes from the diff altogether? You can do this by resetting those files to their contents in the master branch:

```
git reset master go.mod go.sum
git checkout go.mod go.sum
git commit -a -m "Revert changes to go.mod/go.sum"
```